content/influxdb/v2.0/reference/api/influxdb-1x/dbrp.md

```diff
@@ -66,34 +66,3 @@ the DBRP mapping service checks for the specified database and retention policy
 
 - If a mapped bucket exists, data is queried from the mapped bucket.
 - If no mapped bucket exists, InfluxDB returns an error.
```

Comment:
do we need a link to find the curl command from this page back to where it moved to??